src/Consolonia.Gallery/Gallery/GalleryViews/GalleryScrollViewer.axaml.cs

```diff
@@ -61,6 +89,12 @@ public ScrollBarVisibility VerticalScrollVisibility
         }
 
         public List<ScrollBarVisibility> AvailableVisibility { get; }
+
+        public List<string> Cows
```

Comment:
I'm thinking that is miss we are using lists rather than arrays. To support UI update then we could use OvservableCollection or BindingCollection or whatever, but List is worst option